k6/tests/payment100kRegistrationOCW.js

```diff
@@ -0,0 +1,44 @@
+import { check, sleep } from 'k6';
+import loginModel from '../models/login.js';
+import paymentsModel from '../models/payments.js';
+import resetModel from '../models/reset.js';
+
+const paymentsPage = new paymentsModel();
+const resetPage = new resetModel();
+const loginPage = new loginModel();
+
+export const options = {
+  thresholds: {
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+  },
+  vus: 1,
+};
+
+export default function () {
+  // reset db
+  const reset = resetPage.resetDBMockRegistrations(17, '7m');
+  check(reset, {
+    'Reset succesfull status was 202': (r) => r.status == 202,
+  });
+
+  // login
+  const login = loginPage.login();
+  check(login, {
+    'Login succesfull status was 200': (r) => r.status == 201,
+    'Login time is less than 200ms': (r) => {
+      if (r.timings.duration >= 200) {
+        console.log(`Login time was ${r.timings.duration}ms`);
+      }
+      return r.timings.duration < 200;
+    },
+  });
+
+  // do the payment
+  const payment = paymentsPage.createPayment(3);
+  console.log(payment.body);
+  check(payment, {
+    'Payment succesfull status was 202': (r) => r.status == 202,
+  });
```

Comment:
Should we also have a `r.timings.duration <xxx;` here? As in a time it takes before we get back a reponse from the server that the payment has succesfully been initiated

Reply:
What maximal duration is acceptable in this case?

Reply:
I set it to 32s as max allowed time